ocp_resources/endpoints.py

```diff
@@ -0,0 +1,49 @@
+from ocp_resources.constants import TIMEOUT_4MINUTES
+from ocp_resources.resource import NamespacedResource
+
+
+class Endpoints(NamespacedResource):
+    """
+    Endpoints object. API reference:
+    https://docs.openshift.com/container-platform/4.12/rest_api/network_apis/endpoints-v1.html#endpoints-v1
+    """
+
+    api_version = "v1"
+
+    def __init__(
+        self,
+        name=None,
+        namespace=None,
+        client=None,
+        addresses=None,
+        ports=None,
+        teardown=True,
+        privileged_client=None,
+        yaml_file=None,
+        delete_timeout=TIMEOUT_4MINUTES,
+        **kwargs,
+    ):
```

Comment:
please add docstring (mainly for ports and addresses)

Reply:
Added.